unfolder.go

```diff
@@ -130,6 +130,8 @@ func (u *unfolder) handle(writer http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	log.Infof("Message with tid=%v, contentCollectionUuid=%v, collectionType=%v. Done unfolding. Preparing to send messages.", tid, uuid, collectionType)
```

Comment:
without the commas please, so we can accurately search in splunk, not expecting termination characters